communication/zoom-guidelines.md

```diff
@@ -42,3 +42,34 @@ The Zoom licenses are managed by the [CNCF Service Desk](https://github.com/cncf
 ## Escalating and/Reporting a Problem
 
 Issues that cannot be handle via normal moderation can be escalated to the [Kubernetes steering committee](https://github.com/kubernetes/steering). 
+
```

Comment:
Not sure that this file is valid here.

Reply:
Yeah I made a mistake, I'll rebase and submit.